boubleSort/src/main/java/sci/java/RandomName.java

```diff
@@ -0,0 +1,51 @@
+package sci.java;
+
+import java.util.Random;
+
+/**
+ * Clasa pentru a genera un sir de charactere de o anumita lungime (predefinita sau random) dintr-un sir care contine litere alfabetului
+ * returneaza o variabila de tip string
+ */
+
+public class RandomName {
+
+    public String randomName() {
+
+        String caractere;
+        caractere = "abcdefghijklmnopqrstuvwxyz";
```

Comment:
should be a class constant - the string's value is the same for every method call.